src/components/ToggleSwitch/ToggleSwitch.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import Switch from 'react-switch';
+import PropTypes from 'prop-types';
+
+class ToggleSwitch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      toggleCount: 0
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+  //calls notifyChange function on parent
+  handleChange(checked) {
+    this.setState({ checked, toggleCount: this.state.toggleCount + 1 });
+    this.props.notifyChangeToggle(this.props.id, this.state.toggleCount);
+  }
+  //return the right colors for button component
+  getStyle(style) {
+    var colors = {
+      onColor: null,
+      onHandleColor: null,
+      offColor: null,
+      offHandleColor: null
+    };
+    //add style here
+    switch (style) {
+      case 'room':
+        colors = {
+          onColor: '#96a0fa',
+          onHandleColor: '#1f2666',
+          offColor: '#cccccc',
+          offHandleColor: '#666666'
+        };
+        break;
+      case 'device':
+        colors = {
+          onColor: '#96fad5',
+          onHandleColor: '#1f664c',
+          offColor: '#cccccc',
+          offHandleColor: '#666666'
+        };
+        break;
```

Comment:
as eslint suggests: it's better to provide a default case, at least with an error or so: e.g:
```suggestion
      break;
      default:
        throw new Error(`Unknown style id: ${style}`);
```